interop-testing/src/test/java/io/grpc/testing/integration/TestCasesTest.java

```diff
@@ -54,11 +54,27 @@ public void unknownStringThrowsException() {
   @Test
   public void testCaseNamesShouldMapToEnums() {
     // names of testcases as defined in the interop spec
-    String[] testCases = {"empty_unary", "large_unary", "client_streaming", "server_streaming",
-      "ping_pong", "empty_stream", "compute_engine_creds", "service_account_creds",
-      "jwt_token_creds", "oauth2_auth_token", "per_rpc_creds", "custom_metadata",
-      "status_code_and_message", "unimplemented_method", "unimplemented_service",
-      "cancel_after_begin", "cancel_after_first_response", "timeout_on_sleeping_server"};
+    String[] testCases = {
+      "empty_unary",
+      "large_unary",
+      "client_streaming",
+      "server_streaming",
+      "ping_pong",
+      "empty_stream",
+      "compute_engine_creds",
+      "service_account_creds",
+      "jwt_token_creds",
+      "oauth2_auth_token",
+      "per_rpc_creds",
+      "cacheable_unary",
```

Comment:
This is not in the right order. It appears it should be under empty_unary to match the interop spec. It needs to be fixed elsewhere as well.